### PR TITLE
[ONNX] Add xfail into subtests of op consistency and retire fixme

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #100173
* #100172
* #99465

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]

File: test/onnx/test_fx_op_consistency.py [hunk 1]
```diff
@@ -72,13 +71,13 @@
         "ceil", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Ceil")
     ),
-    fixme("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
+    skip("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
```
Comment (BowenBao) [inline test/onnx/test_fx_op_consistency.py:74]: curious why is this skip instead of xfail?

File: test/onnx/test_op_consistency.py [hunk 2]
```diff
@@ -84,27 +84,27 @@
...
         reason=onnx_test_common.reason_onnx_does_not_support("Atan")
     ),
-    fixme("atan2", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Atan", ["f64"])),
+    xfail("atan2", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Atan", ["f64"])),
     skip(
         "ceil", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Ceil")
     ),
-    fixme("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
+    skip("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
```
Comment (BowenBao) [inline test/onnx/test_op_consistency.py:97]: and this

File: test/onnx/test_fx_op_consistency.py (hunk 1 above)
Comment (titaiwangms) [reply test/onnx/test_fx_op_consistency.py]: The way of how xfail vs skip is that we try our best to put on xfail, but if it's not consistently fail, for example, unexpectedly success on few cases, we have to use skip.

But for this one, it's a legacy fixme. Great catch!

**Pull request closed without merging**